tests/cupy_tests/statistics_tests/test_order.py

```diff
@@ -10,113 +9,122 @@
 from cupy import testing
 
 
-_all_interpolations = (
+_all_methods = (
+    # 'inverted_cdf',               # TODO(takagi) Not implemented
+    # 'averaged_inverted_cdf',      # TODO(takagi) Not implemented
+    # 'closest_observation',        # TODO(takagi) Not implemented
+    # 'interpolated_inverted_cdf',  # TODO(takagi) Not implemented
+    # 'hazen',                      # TODO(takagi) Not implemented
+    # 'weibull',                    # TODO(takagi) Not implemented
+    'linear',
+    # 'median_unbiased',            # TODO(takagi) Not implemented
+    # 'normal_unbiased',            # TODO(takagi) Not implemented
     'lower',
     'higher',
     'midpoint',
-    # 'nearest', # TODO(hvy): Not implemented
-    'linear')
+    # 'nearest',                    # TODO(hvy): Not implemented
+)
 
 
-def for_all_interpolations(name='interpolation'):
-    return testing.for_orders(_all_interpolations, name=name)
+def for_all_methods(name='method'):
+    return pytest.mark.parametrize(name, _all_methods)
 
 
-@testing.gpu
-class TestOrder(unittest.TestCase):
+@testing.with_requires('numpy>=1.22.0rc1')
+class TestOrder:
```

Comment:
Skip only `percentile` and `quantile`.

```suggestion
@testing.with_requires('numpy>=1.22.0rc1')
class TestQuantile:
```

and `class TestOrder` for the others.

Then how about applying `@for_all_methods()` to a class?

https://github.com/takagi/cupy/pull/2

Reply:
Thanks, it completely makes sense!